Bottom: 6d7fa97c947bb5efd96d1fff321e7b01b9381f77
Top:    979adf0e0133ec7162c04c19e7b54b6ebf042652
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 21:09:14 +0000

Refresh of well-on-the-way-to-completely

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index dc9c242..bcf5d47 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -556,7 +556,7 @@ object3d_from_board_outline (void)
       /* Link edges orbiting the cylinder top vertex */
       splice (SYM(cylinder_edges[2]), cylinder_edges[1]);
       splice (cylinder_edges[1], SYM(cylinder_edges[1]));
-
+    }
 
       objects = g_list_append (objects, object);
